modules/src/ics02_client/context.rs

```diff
@@ -22,32 +22,25 @@ pub trait ClientKeeper {
     fn store_client_result(&mut self, handler_res: ClientResult) -> Result<(), Error> {
         match handler_res {
             CreateResult(res) => {
-                self.store_client_type(res.client_id.clone(), res.client_type)?;
```

Comment:
Not an issue, just a remark: 
ICS024 [mentions explicitly](https://github.com/cosmos/ics/tree/master/spec/ics-024-host-requirements#path-space) a "path" in the provable store at key `"clients/{identifier}/clientType"`. It seems we will no longer have that path in our provable store. I guess we can still prove client type based on what is stored in client state (at key `"clients/{identifier}/clientState"`).

Reply:
oy! good point!!! added back.